### PR TITLE
eslint fixes.

### PR DESCRIPTION
**Pull request by mrstegeman**
I resisted the urge to re-indent to 2 spaces.

### CODE REVIEW

**Comment by hobinjk** [comment]
I'd like to wait until #5 is merged since it will end up with tons of conflicts otherwise.

File: microblocks-adapter.js
Comment (bromagosa) [inline microblocks-adapter.js]: What's the reason for adding commas after the last element of a dictionary? You only add them sometimes and I can't see the pattern.

File: uprotocol.js
Comment (bromagosa) [inline uprotocol.js]: If possible, I think this file should remain untouched. It was first written for Snap! and it follows its code style, which is Crockford's original JSLint.

File: microblocks-adapter.js
Comment (mrstegeman) [reply microblocks-adapter.js]: Trailing commas make updating objects/arrays easier in the future, as you don't have to remember to add a comma if you're adding a new element at the end.

File: uprotocol.js
Comment (mrstegeman) [reply uprotocol.js]: That's fair.

**Comment by mrstegeman** [comment]
@hobinjk No problem, I can wait.

File: microblocks-adapter.js
Comment (mrstegeman) [reply microblocks-adapter.js]: @bromagosa I only add them when an object/array is split over multiple lines.

Comment (bromagosa) [reply microblocks-adapter.js]: Got it, thanks! :)

**Comment by mrstegeman** [comment]
@hobinjk Updated.

**Review by hobinjk — APPROVED** [review]

**Pull request merged**
